packages/catalog-search/src/SearchSuggestionItem.jsx

```diff
@@ -14,28 +15,44 @@ const SearchSuggestionItem = ({
       suggestionItemHandler(hit);
     }
   };
-
   return (
-    <Link to={url} key={hit.title} className="suggestion-item" onClick={handleLinkDisable}>
-      <div>
-        { /* eslint-disable-next-line react/no-danger, no-underscore-dangle */ }
-        <div dangerouslySetInnerHTML={{ __html: hit._highlightResult.title.value }} />
-        {
-          authoringOrganization && (
-            <div className="badge badge-light ml-3 font-weight-light authoring-org-badge">
-              {authoringOrganization}
+    <div>
+      {isPreQuery ? (
+        <Link to={url} key={hit.title} className="prequery-item pr-4 d-flex flex-column" onClick={handleLinkDisable}>
+          <div className="d-flex align-items-center justify-content-start">
+            <Image className="prequery-image mr-2" src={hit.card_image_url} />
+            <div className="d-flex flex-column">
```

Comment:
are these two rendered components worth pulling out into their own separate components?

Reply:
I think that the components are similar/small enough to keep as one. lmk if you feel strongly otherwise and I can follow-up with another change.